README.md

```diff
@@ -9,7 +9,7 @@ Forecast    'http://api.openweathermap.org/data/2.5/forecast?q={city}&cnt={days_
 * Define and call a [function](https://www.w3schools.com/python/python_functions.asp)
 * Format strings using [f-strings](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python)
 * Use [urllib](https://docs.python.org/3/howto/urllib2.html) to make requests to the API
-* Create a [virtualenv](https://docs.python.org/3/library/venv.html) for managing dependencies
+* Create a [virtualenv](https://docs.python.org/3/library/venv.html) for managing dependencies // More virtual environment examples [here](/HowToVenv)
```

Comment:
Does this link work? 👀 Or does it need to point at the readme in /HowToVenv/README.md?

Reply:
> Looks mainly good! I added some thoughts. Could you do something to the commit history as well, unless you plan on squashing it in the merge?

I will take a second look for that. Tried to squash them, but things went horribly wrong :D

Reply:
> Does this link work? 👀 Or does it need to point at the readme in /HowToVenv/README.md?

Yep, it will point to the Folder and folder itself shows the README.md like the Repo's front page readme

Reply:
I feel ya! You could try 
`git reset HEAD~5` (or how many steps backward you wanna go)
`git add .`
`git commit -m "My new fancy commit`

Reply:
I can try that, tried with rebase after midnight... soo maybe with more fresh mind it will work :D